src/t2iapi/context/context_responses.proto

```diff
@@ -33,10 +33,10 @@ message GetSupportedContextTypesResponse {
 }
 
 /*
-Response which contains Identification including a list of InstanceIdentifier elements
-which identify an ensemble and thus enables the caller to distinguish ensembles from each other.
+Response which contains a list of lists with pm:Identification elements. Each list if pm:Identification elements
```

Comment:
typo: Each list **of** pm:Identification elements...

Reply:
fixed